wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -50,7 +51,6 @@ import           Cardano.Wallet.API.V1.Generic (gparseJsend, gtoJsend)
 data WalletError =
       NotEnoughMoney { weNeedMore :: !Int }
     | OutputIsRedeem { weAddress :: !Text }
-    | SomeOtherError { weFoo :: !Text, weBar :: !Int }
```

Comment:
Yes, as you guessed this `SomeOtherError` was used just as a witness it was possible to generate jsend-style errors with more than 1 field in the `diagnostic` section, and encode everything using `generics-sop`. In practice, it's for debugging purposes only and I agree it's better to remove it.